src/command/config/profile.rs

```diff
@@ -38,7 +41,7 @@ impl Profile {
     pub fn run(&self) -> Result<()> {
         match &self.command {
             Command::List => {
-                let profiles = config::Profile::list()?;
+                let profiles = config::Profile::list().context("Could not list profiles.")?;
```

Comment:
So based on my knowledge of `.context` which is very little...

This in essence would still print the underlying error from `.list` but it would print the "Context" of that error before? 

Like the example on [this page](https://docs.rs/anyhow/1.0.32/anyhow/trait.Context.html#example)

```
Error: Could not list profiles

Caused by:
    Whatever error was returned by .list
```

Reply:
yes! that's exactly right. another way to think about it is that you are "wrapping" the error that comes from the library with your own description of what happened. this is super useful for us and i'd say almost all `?` values that we use should have `context` applied. (i'm not sure if i do it here at all, but `with_context(|_| format!("hi {}", "there"))` is something you could also do :smile: